### ISSUE TITLE
bulk edit of ascents

### ISSUE THREAD

**Issue by scd** [theCrag/website#2787]
Adding proper bouldering types is means people what to change their historical ticks. 

It would be good to be able to apply a change across many ascents and/or a bulk edit interface. I think the bulk edit is probably not worthwhile, but applying a change across multiple ticks would save users a lot of time.

**Comment by brendanheywood** [comment]
+10 for adding bulk edit of ascents. I can't remember how many times I've opened up a large batch of ascents 1 in each tab and edited them all in parallel. I've done that for various reasons in particular dates and 'who'. Solving this via bulk edit is a much better solution rather than a more specific fix just for the boulder tick types.

**Comment by lordyavin** [comment]
+30

**Comment by brendanheywood** [comment]
+1 from @quaestor (via heart)

**Comment by brendanheywood** [comment]
+1 via duplicate https://github.com/theCrag/website/issues/1835

**Comment by johnpitcairn** [comment]
A couple of years ago I thought that perhaps exporting the logbook, editing (search & replace in excel or by script or whatever) then re-importing might be a good way to accomplish things like updating tick types. 

That would just need the import functions to be able to import its own export format.

**Comment by brendanheywood** [comment]
@johnpitcairn was this due to https://github.com/theCrag/website/issues/2225 / https://github.com/theCrag/website/issues/1307 (duplicates which I just fixed too) or was there something else busted?

Please test on https://brendan.thecrag.com/ and let me know if you have issues

**Comment by johnpitcairn** [comment]
No, at the time I wanted to be able to re-import the CSV export after I had manipulated it. That still isn't possible, is it?

**Comment by lordyavin** [comment]
Hi @brendanheywood, would like to test the bulk edit feature but couldn't log in on the test server.

**Comment by brendanheywood** [comment]
There is no bulk edit feature yet.

There is a potential workaround: the logbook csv download, edit offline, and then re-import which *should* work but I've never tested it myself. I believed it was technically blocked by the two bugs I mentioned above (csv column names where mixed up) which were super easy and I just fixed them. But the logbook import process is really clunky as we've never invested the time to get this into shape. On top of that there is a daemon on my dev box which is broken so I can't even test it.

When you say you can't login, more specific details? I'm using it right now without issue (apart from above)

**Comment by Mdemaillard** [comment]
Duplicate of #2030
